Dockerfile

```diff
@@ -1,3 +1,6 @@
-FROM italia/publiccode-parser-go:latest
+FROM bfabio/publiccode-parser-go:latest
```

Comment:
I'd push this image under italia org using an appropriate tag

Reply:
Agreed! I'd tag it as `alpha`.